### ISSUE TITLE
Can't install

### ISSUE THREAD

**Issue by harchus** [OpenLiveWriter/OpenLiveWriter#20]
Installation has failed.

Unable to write to C:/ProgramData/xxx - IT policies may be restricting access to this folder.

Windows 10 TH2 latest update.

**Comment by RobDolinMS** [comment]
@harchus are you running as a user that may have some IT policy restrictions?

**Comment by RobDolinMS** [comment]
In your reply, please tag @willduff to take a look.

**Comment by RobDolinMS** [comment]
We should add this to the FAQ.

**Comment by Crypt32** [comment]
Facing the same issue today.

No executable/dll shall be installed in the user's profile. They should reside in the Program Files. Because too many items should be whitelisted (because DLL checking is enabled as well).

**Comment by pekiZG** [comment]
@willduff I would suggest %AppData% usage to avoid this problem.
Would that be smart?

**Comment by paulcbetts** [comment]
It's surprising that both %LocalAppData% and the ProgramData fallback are being blocked - usually the latter works.

**Comment by fsacer** [comment]
Can you review chnage for a wiki repo https://github.com/fsacer/OpenLiveWriter.Wiki where I added question about the installer to FAQ?

**Comment by harchus** [comment]
Personal computer, no IT, the thing is even I got that error OLW installed successfully. :/

**Comment by Crypt32** [comment]
@pekiZG no. AppData is BAD solution. Like I mentioned earlier the only right solution (from security and management perspective) is:
1) all binaries shall be installed in the Program Files folder.
2) non-executable app-specific files must be stored in the user profile.
Other options are insecure and shall be provided as an option.

**Comment by datsomiro** [comment]
hy guys
after downloading github for desktop unable to install getting Unable to write to C:/ProgramData/xxx - IT policies may be restricting access to this folder.
windows 10 latest update
any help ?

**Comment by shanselman** [comment]
Thanks for reporting this and for helping Open Live Writer over the years. We are doing an issue-bankruptcy cleanup so we can make the repository maintainable again and focus on current, actionable work. This older issue is being closed as part of that cleanup, not because it was unimportant. If this still reproduces in the current code or is still relevant to the revitalization work, please open a fresh issue with current details and we will triage it going forward.
